__device-tests__/helpers/utils.js

```diff
@@ -56,6 +64,18 @@ const setupDriver = async () => {
 		desiredCaps = _.clone( android8 );
 		if ( isLocalEnvironment() ) {
 			desiredCaps.app = localAndroidAppPath;
+			try {
+				const androidVersion = childProcess
+					.execSync( 'adb shell getprop ro.build.version.release' )
+					.toString()
+					.replace( /^\s+|\s+$/g, '' );
+				if ( ! isNaN( androidVersion ) ) {
+					desiredCaps.platformVersion = androidVersion;
```

Comment:
What's the result of this if there are multiple running Android devices with different versions? Do you think it's a good idea to accept maybe an ENV variable for this? I was also looking into the use of the `emulator` cli for firing up the Android emulator if it's not running?  WDYT?

Reply:
Also another thing to note is that the specification of the Android platform version is more a requirement for the CI environment, locally if this isn't specified it'll just use the first running one it picks up

Reply:
So you mean we could just clear it?
```
desiredCaps.platformVersion = undefined;
```

Reply:
Well, I think that'll still have the key in there and I'm not actually sure how it'd respond to that. `delete desiredCaps.platformVersion` would be better if we want it to just pick up any running emulator.